lib/cassandra_migrations/railtie/initializer.rb

```diff
@@ -11,20 +11,24 @@
 
 module CassandraMigrations
 
+  if defined?(Spring)
```

Comment:
I'm not a spring user, so I am trusting you on that. In respect to the forking behavior with using Passenger, I noticed we lost the logging of the connection being started/restarted. No need to distinguish both, but it would be good to keep the log line.

About the readme, I am always in favor to docs improvement. I was wondering though that the readme is already kind of long. What do you think about adding a wiki page and linking it on the readme?